static/app/stores/groupingStore.tsx

```diff
@@ -114,7 +116,10 @@ type InternalDefinition = {
   api: Client;
 };
 
-interface GroupingStoreDefinition extends StoreDefinition, InternalDefinition {
+interface GroupingStoreDefinition
+  extends StoreDefinition,
+    CommonStoreDefinition<State>,
```

Comment:
Does this need to extend StoreDefinition if CommonStoreDefinition already extends it?

Reply:
Nope! Good catch, I went over all the other stores to see if we did this someplace else and looks like this was the only case!